install.ps1

```diff
@@ -444,11 +444,12 @@ if ($Distro) {
 } else {
     $Distribution = Get-WslDistribution -Default | Select-Object -first 1
     if (-not $Distribution) {
-        Write-Error "!!! $Distro is not currently installed, and you do not have a default distribution. Refusing to continue."
+        Write-Error "!!! You do not have a default distribution. Refusing to continue."
         exit
     }
     Write-Debug "--- No distro specified, using your default distro $($Distribution.Name)"
 }
+$Distro = $($Distribution.Name)
```

Comment:
I think it would be better to consistently use `$($Distribution.Name)`, or update all instances of `$($Distribution.Name)` to `$Distro` after this line (I prefer the former, but I don't mind either way really).